<?xml version="1.0"?>
<launch>

  <include file="$(find realsense2_camera)/launch/demo_pointcloud.launch" />
 <node name="pcl4" pkg="realsense_devel" type="pcl4"/>

</launch> 
